modules/monitoring-accessing-third-party-uis-using-the-cli.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-uis.adoc
+
+[id="accessing-third-party-uis-using-the-cli_{context}"]
+= Accessing third-party monitoring UIs by using the CLI
+
+You can obtain URLs for the Prometheus, Alertmanager, and Grafana web UIs by using the OpenShift CLI (`oc`) tool.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Run the following to list routes for the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get routes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                HOST/PORT                                                     ...
+alertmanager-main   alertmanager-main-openshift-monitoring.apps._url_.openshift.com ...
+grafana             grafana-openshift-monitoring.apps._url_.openshift.com           ...
+prometheus-k8s      prometheus-k8s-openshift-monitoring.apps._url_.openshift.com    ...
+thanos-querier      thanos-querier-openshift-monitoring.apps._url_.openshift.com    ...
+----
+
+. Prepend `https://` to the `HOST/PORT` addresses that are output by the preceding command. In this example, the resulting URL for Alertmanager is as follows:
```

Comment:
no need to prepend `https://` to the `HOST/PORT` addresses, since it will redirect to https, we can keep it unchanged, it is not wrong with the prepended `https://`
```# oc -n openshift-monitoring get routes
NAME                HOST/PORT                                                                                PATH   SERVICES            PORT    TERMINATION          WILDCARD
alertmanager-main   alertmanager-main-openshift-monitoring.apps.qe-groupd-0923.qe.devcluster.openshift.com          alertmanager-main   web     reencrypt/Redirect   None
```

Reply:
@juzhao thank you for your review and for this suggestion.

I have now removed this step from the module completely.

I have now resolved this on my local clone and will push the change to my fork shortly.